hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -322,6 +326,16 @@ public Response put(
     }
   }
 
+  private static boolean hasTrailingSlash(String keyPath) {
+    return keyPath.charAt(keyPath.length() - 1) == '/';
+  }
+
+  private void createDirectory(String volumeName,
+      String bucketName, String keyPath) throws IOException {
+    getClientProtocol()
+        .createDirectory(volumeName, bucketName, keyPath);
+  }
```

Comment:
In the current change set this helper method is only called once. I think it can be removed.

Reply:
Done.